### PR TITLE
feat: fixed toast ui

### PR DESCRIPTION
**Pull request by Oluwaseyi3**
## Problem

The icon wasn't aligned with the text  in the toast UI. It also needed some padding 

## Changes Made

I worked on the UI based the Figma model and reconciled the differences with Tailwind CSS

## Screenshots

[Include screenshots, if relevant, to help reviewers understand the changes you made.]

toast

## Reviewers

@mlabouardy

### CODE REVIEW

File: dashboard/components/toast/Toast.tsx [hunk 1]
```diff
@@ -42,10 +42,10 @@ function Toast({ hasError, title, message, dismissToast }: ToastProp) {
               <CheckIcon width={24} height={24} />
             )}
           </div>
-          <div>
-            <p className="text-sm font-medium">{title}</p>
+          <div className="p-3">
+            <p className="pb-2 text-xl font-medium">{title}</p>
```
Comment (mlabouardy) [inline dashboard/components/toast/Toast.tsx:46]: can you revert to text-sm as the title is quite huge :) 
image

**Review by mlabouardy — CHANGES_REQUESTED** [review]
Thanks @Oluwaseyi3, a small change and we should be good :)

File: dashboard/components/toast/Toast.tsx (hunk 1 above)
Comment (Oluwaseyi3) [reply dashboard/components/toast/Toast.tsx:46]: Alright. On it Mohamed!

Comment (Oluwaseyi3) [reply dashboard/components/toast/Toast.tsx]: Hi @mlabouardy I set it to text-lg and it looks perfectly okay for a title. Just a bit bigger than the text body. You can check it out. 



hh

If you still want it to be reviewed pls don't hesitate to notify me

Comment (mlabouardy) [reply dashboard/components/toast/Toast.tsx]: all good then :)

**Review by mlabouardy — APPROVED** [review]
Thanks for the changes @Oluwaseyi3 :)

**Comment by mlabouardy** [comment]
> Thanks for the changes @Oluwaseyi3 :)

seems like CI failed due to package.json changes, could you check that @Oluwaseyi3 
https://github.com/tailwarden/komiser/actions/runs/5334667306/jobs/9672597140?pr=862

**Comment by Oluwaseyi3** [comment]
> > Thanks for the changes @Oluwaseyi3 :)
> 
> seems like CI failed due to package.json changes, could you check that @Oluwaseyi3 https://github.com/tailwarden/komiser/actions/runs/5334667306/jobs/9672597140?pr=862

Alright. Will correct that asap

**Comment by Oluwaseyi3** [comment]
> > > Thanks for the changes @Oluwaseyi3 :)
> > 
> > 
> > seems like CI failed due to package.json changes, could you check that @Oluwaseyi3 https://github.com/tailwarden/komiser/actions/runs/5334667306/jobs/9672597140?pr=862
> 
> Alright. Will correct that asap



> > > Thanks for the changes @Oluwaseyi3 :)
> > 
> > 
> > seems like CI failed due to package.json changes, could you check that @Oluwaseyi3 https://github.com/tailwarden/komiser/actions/runs/5334667306/jobs/9672597140?pr=862
> 
> Alright. Will correct that asap

@mlabouardy Pls you can check it out now. I updated the package.json. Let me know if the test cli passes

**Comment by Oluwaseyi3** [comment]
> > > > Thanks for the changes @Oluwaseyi3 :)
> > > 
> > > 
> > > seems like CI failed due to package.json changes, could you check that @Oluwaseyi3 https://github.com/tailwarden/komiser/actions/runs/5334667306/jobs/9672597140?pr=862
> > 
> > 
> > Alright. Will correct that asap
> 
> > > > Thanks for the changes @Oluwaseyi3 :)
> > > 
> > > 
> > > seems like CI failed due to package.json changes, could you check that @Oluwaseyi3 https://github.com/tailwarden/komiser/actions/runs/5334667306/jobs/9672597140?pr=862
> > 
> > 
> > Alright. Will correct that asap
> 
> @mlabouardy Pls you can check it out now. I updated the package.json. Let me know if the test cli passes




Hi @mlabouardy just noticed it failed once more. I’ve installed a new package.lock. This one is in sync with the develop branch. This would work now

**Review by ShubhamPalriwala — CHANGES_REQUESTED** [review]
Hey @Oluwaseyi3, I can still see a lot of package-lock.json changes in the git diff hence why the build is failing due to dependency mismatch.

One suggestion for a cleaner and quicker fix would be to remove the `package-lock.json` from the git tracking of this PR and just add and commit the `dashboard/components/toast/Toast.tsx` file, this should solve it all at once and the PR should be good to merge then

**Comment by Oluwaseyi3** [comment]
> Hey @Oluwaseyi3, I can still see a lot of package-lock.json changes in the git diff hence why the build is failing due to dependency mismatch.
> 
> One suggestion for a cleaner and quicker fix would be to remove the `package-lock.json` from the git tracking of this PR and just add and commit the `dashboard/components/toast/Toast.tsx` file, this should solve it all at once and the PR should be good to merge then

On it

**Comment by Oluwaseyi3** [comment]
> Hey @Oluwaseyi3, I can still see a lot of package-lock.json changes in the git diff hence why the build is failing due to dependency mismatch.
> 
> One suggestion for a cleaner and quicker fix would be to remove the `package-lock.json` from the git tracking of this PR and just add and commit the `dashboard/components/toast/Toast.tsx` file, this should solve it all at once and the PR should be good to merge then

All done @ShubhamPalriwala

**Pull request merged**